libs/blocks/commerce/commerce.js

```diff
@@ -91,6 +114,14 @@ export const decorateOfferDetails = async (el, of, searchParams) => {
   });
   const checkoutUrl = buildCheckoutButton(checkoutLink, options);
   checkoutUrl.target = '_blank';
+  const clearButton = buildClearButton();
+
+  clearButton.addEventListener('click', () => {
```

Comment:
If you already have `buildClearButton` you could have the event listener live in there too